python-package/lightgbm/callback.py

```diff
@@ -215,6 +221,10 @@ def _callback(env):
         if not enabled[0]:
             return
         for i in range_(len(env.evaluation_result_list)):
+            metric_key = env.evaluation_result_list[i][1]
+            if env.evaluation_result_list[i][0] == "cv_agg" and metric_key.split(" ")[0] == "train":
+                # train data for lgb.cv
+                continue
```

Comment:
What about the case for `lgb.train` `env.evaluation_result_list[i][0] == env.model._train_data_name:`? Seems that it was lost in the last commit.